### PR TITLE
[Doc] improve README and other documents

### PR DESCRIPTION
**Pull request by EricJoy2048**
1. Improve README.
2. Add SeaTunnel Engine README
3. Improve quick start
4. Improve connector status document
5. Add SeaTunnel Engine to doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
-SeaTunnel will do its best to solve the problems that may be encountered in the synchronization of massive data:
+SeaTunnel focuses on data integration and data synchronization, and is mainly designed to solve common problems in the field of data integration:
 
-- Data loss and duplication
-- Task accumulation and delay
-- Low throughput
-- Long cycle to be applied in the production environment
-- Lack of application running status monitoring
+- Various data sources: There are hundreds of commonly-used data sources of which versions are incompatible. With the emergence of new technologies, more data sources are appearing. It is difficult for users to find a tool that can fully and quickly support these data sources.
+- Complex synchronization scenarios: Data synchronization needs to support various synchronization scenarios such as offline-full synchronization, offline- incremental synchronization, CDC, real-time synchronization, and full database synchronization.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- Complex synchronization scenarios: Data synchronization needs to support various synchronization scenarios such as offline-full synchronization, offline-incremental synchronization, CDC, real-time synchronization, and full database synchronization.
```

File: README.md [hunk 2]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
+- High demand in resource: Existing data integration and data synchronization tools often require vast computing resources or JDBC connection resources to complete real-time synchronization of massive small tables. This has increased the burden on enterprises to a certain extent.
+- Lack of quality and monitoring: Data integration and synchronization processes often experience loss or duplication of data. The synchronization process lacks monitoring, and it is impossible to intuitively understand the real-situation of the data during the task process.
+- Complex technology stack: The technology components used by enterprises are different, and users need to develop corresponding synchronization programs for different components to complete data integration.
+- Difficulty in management and maintenance: Limited to different underlying technology components (Flink/Spark) , offline synchronization and real-time synchronization often have be developed and managed separately, which increases thedifficulty of the management and maintainance.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- Difficulty in management and maintenance: Limited to different underlying technology components (Flink/Spark) , offline synchronization and real-time synchronization often have be developed and managed separately, which increases the difficulty of the management and maintainance.
```

File: README.md [hunk 3]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
 
-## SeaTunnel use scenarios
+## Features of SeaTunnel
 
-- Mass data synchronization
-- Mass data integration
-- ETL with massive data
-- Mass data aggregation
-- Multi-source data processing
+- Rich and extensible Connector: SeaTunnel provides a Connector API that does not depend on a specific execution engine. Connectors (Source, Transform, Sink) developed based on this API can run On many different engines, such as SeaTunnel Engine, Flink, Spark that are currently supported.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- Rich and extensible Connector: SeaTunnel provides a Connector API that does not depend on a specific execution engine. Connectors (Source, Transform, Sink) developed based on this API can run on many different engines, such as SeaTunnel Engine, Flink, Spark that are currently supported.
```

File: README.md [hunk 4]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
+- Connector plug-in: The plug-in design allows users to easily develop their own Connector and integrate it into the SeaTunnel project. Currently, SeaTunnel has supported more than 70 Connectors, and the number is surging. There is the list of the currently-supported connectors: xxxxxxx, and t he list of planned connectors: xxxxxxx.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- Connector plugin: The plugin design allows users to easily develop their own Connector and integrate it into the SeaTunnel project. Currently, SeaTunnel has supported more than 70 Connectors, and the number is surging. There is the list of the currently supported connectors: xxxxxxx, and the list of planned connectors: xxxxxxx.
```

Comment (TaoZex) [reply README.md]: Using `plug-in` is right, but `plugin` is better.

File: README.md [hunk 5]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
+- Batch-stream integration: Connectors developed based on SeaTunnel Connector API are perfectly compatible with offline synchronization, real-time synchronization, full- synchronization, incremental synchronization and other scenarios. It greatly reduces the difficulty of managing data integration tasks.
+- Support distributed snapshot algorithm to ensure data consistency.
+- Multi-engine support: SeaTunnel uses SeaTunnel Engine for data synchronization by default. At the same time, SeaTunnel also supports the use of Flink or Spark as the execution engine of the Connector to adapt to the existing technical components of the enterprise. SeaTunnel supports multiple versions of Spark and Flink.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- Multi-engine support: SeaTunnel uses SeaTunnel Engine for data synchronization by default. At the same time, SeaTunnel also supports the use of Flink or Spark as the execution engine of the Connector to adapt to the existing technical components of the enterprise. In addition, SeaTunnel supports multiple versions of Spark and Flink.
```

File: README.md [hunk 6]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
+- JDBC multiplexing, database log multi-table parsing: SeaTunnel supports multi-table or whole database synchronization, which solves the problem of over- JDBC connections; supports multi-table or whole database log reading and parsing, which solves the need for CDC multi-table synchronization scenarios Problems with repeated reading and parsing of logs.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- JDBC multiplexing, database log multi-table parsing: SeaTunnel supports multi-table or whole database synchronization, which solves the problem of over-JDBC connections; supports multi-table or whole database log reading and parsing, which solves the need for CDC multi-table synchronization scenarios problems with repeated reading and parsing of logs.
```

File: README.md [hunk 7]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
+- High throughput and low latency: SeaTunnel supports parallel reading and writing, providing stable and reliable data synchronization capabilities with high throughput and low latency.
+- Perfect real-time monitoring: SeaTunnel supports detailed monitoring information of each step in the data synchronization process, allowing users to easily understand the number of data, data size, QPS and other information read and written by the synchronization task.
+- Two job development methods are supported: coding and canvas design: The SeaTunnel web project https://github.com/apache/incubator-seatunnel-web provides visual management of jobs, scheduling, running and monitoring capabilities.
```
Comment (TaoZex) [inline README.md]:
```suggestion
- Two job development methods are supported: coding and canvas design. The SeaTunnel web project https://github.com/apache/incubator-seatunnel-web provides visual management of jobs, scheduling, running and monitoring capabilities.
```

File: docs/en/start-v2/locally/quick-start-flink.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+You could start the application by the following commands
+
+```shell
+cd "apache-seatunnel-incubating-${version}"
+./bin/start-seatunnel-flink-connector-v2.sh --config ./config/seatunnel.streaming.conf.template
+
+```
+
+**See The Output**: When you run the command, you could see its output in your console, You can think this
```
Comment (TaoZex) [inline docs/en/start-v2/locally/quick-start-flink.md]:
```suggestion
**See The Output**: When you run the command, you could see its output in your console. You can think this
```

File: docs/en/start-v2/locally/quick-start-seatunnel-engine.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+You could start the application by the following commands
+
+```shell
+cd "apache-seatunnel-incubating-${version}"
+./bin/seatunnel.sh --config ./config/seatunnel.streaming.conf.template -e local
+
+```
+
+**See The Output**: When you run the command, you could see its output in your console, You can think this
```
Comment (TaoZex) [inline docs/en/start-v2/locally/quick-start-seatunnel-engine.md]:
```suggestion
**See The Output**: When you run the command, you could see its output in your console. You can think this
```

File: docs/en/start-v2/locally/quick-start-spark.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+```shell
+cd "apache-seatunnel-incubating-${version}"
+./bin/start-seatunnel-spark-connector-v2.sh \
+--master local[4] \
+--deploy-mode client \
+--config ./config/seatunnel.streaming.conf.template
+```
+
+**See The Output**: When you run the command, you could see its output in your console, You can think this
```
Comment (TaoZex) [inline docs/en/start-v2/locally/quick-start-spark.md]:
```suggestion
**See The Output**: When you run the command, you could see its output in your console. You can think this
```

File: README.md [hunk 11]
```diff
@@ -19,49 +19,43 @@ been used in the production of nearly 100 companies.
...
-## SeaTunnel use scenarios
+## Features of SeaTunnel
 
-- Mass data synchronization
-- Mass data integration
-- ETL with massive data
-- Mass data aggregation
-- Multi-source data processing
+- Rich and extensible Connector: SeaTunnel provides a Connector API that does not depend on a specific execution engine. Connectors (Source, Transform, Sink) developed based on this API can run on many different engines, such as SeaTunnel Engine, Flink, Spark that are currently supported.
+- Connector plugin: The plugin design allows users to easily develop their own Connector and integrate it into the SeaTunnel project. Currently, SeaTunnel has supported more than 70 Connectors, and the number is surging. There is the list of the currently supported connectors: xxxxxxx, and the list of planned connectors: xxxxxxx.
```
Comment (CalvinKirs) [inline README.md]: We can link connector-status and connector-support plan to here

**Review by PraveenNanda124 — COMMENTED** [review]
Good work

**Review by TaoZex — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**